tests/unit/systems/config_file/init_test.py

```diff
@@ -0,0 +1,32 @@
+import os
+import tempfile
+import textwrap
+
+import dusty.constants
+from dusty.systems import config_file
+from ....testcases import DustyTestCase
+
+class TestConfigFileSystem(DustyTestCase):
+    def setUp(self):
+        super(TestConfigFileSystem, self).setUp()
+        self.spec_output = textwrap.dedent("""\
+        # BEGIN section for Dusty
+        127.0.0.1 local.gc.com
+        127.0.0.2 local-api.gc.com
+        # END section for Dusty
+        """)
+        self.non_spec_starter = "127.0.0.1 some-host.local.com\n"
+
+    def test_remove_current_dusty_config_from_blank(self):
+        result = config_file.remove_current_dusty_config(self.spec_output)
+        self.assertEqual(result, "")
+
+    def test_remove_current_dusty_config_from_starting(self):
+        result = config_file.remove_current_dusty_config(self.non_spec_starter + self.spec_output)
+        self.assertEqual(result, self.non_spec_starter)
```

Comment:
Probably worth making sure that stuff after the Dusty section also doesn't get blown away